dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/logging/LogUtils.java

```diff
@@ -264,36 +264,64 @@ protected static Logger createLogger(final Class<?> cls,
                 try {
                     b = BundleUtils.getBundle(cls, bundleName);
                 } catch (MissingResourceException rex) {
-                    //ignore
                 }
             }
             if (b != null) {
                 b.getLocale();
             }
 
             if (loggerClass != null) {
-                try {
-                    Constructor<?> cns = loggerClass.getConstructor(String.class, String.class);
-                    if (name == null) {
+                // Default logger of Liberty is WsLogger. 
+                // CXFLogger is inherited from this WsLogger class to preserve same behavior
+                // except the one that we aimed to change. Which is to add class name to log records
+                if (loggerClass.equals(WsLogger.class)) {
```

Comment:
I'm concerned that the direct reference to our internal logging API will cause CDNFEs when our trace isn't configured, but they have their own logger configured. The logic is sound, I just wonder if we should be using reflection rather then a direct import. I think the only way to know for sure is to test this ourselves.